### ISSUE TITLE
Output of the CCT classifier

### ISSUE THREAD

**Issue by enrico310786** [SHI-Labs/Compact-Transformers#63]
Hi,

i am a little confuse about the output of the CCT. If I have a classification task with n possible classes, are the outputs the logits of each class? Thus to find the respective probabilities i have to apply a softmax function, or are the outputs the probabilities?

Thanks in advance

**Comment by alihassanijr** [comment]
Hi and thank you for your interest.

I apologize for noticing this issue so late.
If you're referring to the final outputs, it's similar to most image classification models in pytorch:
Softmax is not applied to the output, because the cross entropy loss does that for you when training.
So in essence you would need to manually apply it for inference.

I hope this answers your question.

**Comment by stevenwalton** [comment]
Closing due to inactivity. If this is a mistake please reopen.
